ktlint-reporter-sarif/build.gradle

```diff
@@ -1,18 +0,0 @@
-plugins {
-    id 'ktlint-kotlin-common'
-    id 'ktlint-publication'
-}
-
-dependencies {
-  implementation projects.ktlintCore
-  implementation libs.sarif4k
-
-  testImplementation libs.junit5
-  testImplementation libs.assertj
-}
-
-jar {
-    manifest {
-        attributes 'Implementation-Version': project.property('VERSION_NAME')
-    }
-}
```

Comment:
I'm removing the `jar` block as it is showing in `ktlint` module, right?

https://github.com/Goooler/ktlint/blob/2f7587351e8e57d30148683ab4fc0ea9d3c12a49/ktlint/build.gradle.kts#L11-L16

Reply:
@romtsn @shashachu I can not judge this.